tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_test_resources.py

```diff
@@ -266,11 +266,10 @@ def cleanup(self) -> None:
     @functools.lru_cache(None)
     def default_backend_service(self) -> str:
         """Returns default backend service URL."""
-        self.td.load_backend_service()
-        return self.td.backend_service.url
+        return self.td.make_resource_name(self.td.BACKEND_SERVICE_NAME)
 
     @functools.lru_cache(None)
     def alternative_backend_service(self) -> str:
         """Returns alternative backend service URL."""
-        self.td.load_alternative_backend_service()
```

Comment:
This change is good. If you want this in, let's move it to another PR?

Reply:
Created as https://github.com/grpc/grpc/pull/26713